### PR TITLE
NScala-time instances

### PR DESCRIPTION
**Pull request by folone**
Added simple instances for the [nscala-time](https://github.com/nscala-time/nscala-time) project.

### CODE REVIEW

File: nscala-time/main/scala/package.scala [hunk 1]
```diff
@@ -0,0 +1,3 @@
+package scalaz.contrib
+
+package object nscala_time extends DurationInstances with PeriodInstances
```
Comment (larsrh) [inline nscala-time/main/scala/package.scala:3]: That underscore disturbs me. It's the name of the original package, though.

**Comment by larsrh** [comment]
Thank you!

File: nscala-time/main/scala/package.scala (hunk 1 above)
Comment (folone) [inline nscala-time/main/scala/package.scala:3]: It disturbs me a bit too. I originally called the object `nscalatime`, but then decided to stick to the convention in this library (name packages the same way as packages in 3rd-party libs). If you think, `nscalatime` is a better name for the package, I totally agree.

Comment (larsrh) [inline nscala-time/main/scala/package.scala:3]: Consistency trumps beauty :-)

**Pull request merged**